### PR TITLE
Fixes/return array when parsing response

### PR DESCRIPTION
**Pull request by mauro-oto**
Hey @etagwerker, @schmierkov, 

This PR changes `parse_results_table` in the `BaseClient` class to always return an array. 

I also changed the hash keys `Epak::Client#get_shipping_rates` receives and deprecated (removed entirely) `check_operation` as it's no longer useful since the introduction of `get_operation_codes`. 

Please check it out, thanks!

### CODE REVIEW

File: lib/oca-epak/epak/client.rb [hunk 1]
```diff
@@ -14,23 +14,7 @@ def check_credentials
...
         response = client.call(method, message: opts)
-        parse_results_table(response, method)[:existe] == "1"
-      end
-
-      # Checks whether the operation is valid
-      #
-      # @param [String] Client's CUIT
-      # @param [String] Operation Type
-      # @return [Boolean]
-      def check_operation(cuit, op)
```
Comment (etagwerker) [inline lib/oca-epak/epak/client.rb]: @mauro-oto Just curious, why did you remove `#check_operation`?

Comment (mauro-oto) [inline lib/oca-epak/epak/client.rb]: @etagwerker I always felt it was a poor implementation, it asked the Oca API for a shipping rate check using random details, because the `get_operation_codes` method was not available at the time. Since that method is now available, it doesn't make much sense to have that now. Another solution would be to re-implement `check_operation` using `get_operation_codes` by checking the operations in the resulting array, which would be much better but I don't see much use for it.

Comment (etagwerker) [inline lib/oca-epak/epak/client.rb]: @mauro-oto got it! :+1:
